packages/dashboard/src/app/views/shared/pageHeading.js

```diff
@@ -70,7 +72,18 @@ const PageHeading = ({
   showSearch,
   handleSearchChange,
   searchValue = '',
+  clearSearch = noop,
 }) => {
+  const handleClearSearch = useCallback(
+    (evt) => {
+      // evt is null when user presses the clear button
+      if (evt === null) {
+        clearSearch();
+      }
+    },
+    [clearSearch]
```

Comment:
💭 this `evt === null` thing is a little smelly to me - feels like we should update the api to pass a separate `onClear` event handler.

But that's not that important and can always be done in another ticket

Reply:
nah that makes sense. will update

Reply:
https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/9667